integrated_channels/cornerstone/admin/__init__.py

```diff
@@ -66,6 +66,8 @@ class CornerstoneEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
 
     readonly_fields = (
         "enterprise_customer_name",
+        "session_token",
```

Comment:
the relevant model fields https://github.com/edx/edx-enterprise/blob/master/integrated_channels/cornerstone/models.py#L104